src/vector/index.js

```diff
@@ -274,21 +277,21 @@ async function loadApp() {
                 // in case the Tinter.tint() in MatrixChat fires before the
                 // CSS has actually loaded (which in practice happens)...
 
-                // This if fixes Tinter.setTheme to not fire on Firefox
+                // This if fixes setTheme to not fire on Firefox
```

Comment:
```suggestion
                // This fixes setTheme to not fire on Firefox
```

Reply:
it is an if though?

Reply:
Oh, I see, it's far enough from the code that I didn't register it... I guess it's fine then.